CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Note: Will not be required anymore. I added this in a separate PR.

Reply:
You mean we don't need to add ## [Unreleased] part for any new entries in CHANGELOG?

Reply:
No no, I just added `[Unreleased]` section in last PR so no need for this here, you can now just add a CHANGELOG as usual. Sorry if I confused you here.